[2017-08-07 20:45:13,557] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for 13b92fbb-d2a4-4327-a685-0a5ec356dc3b failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> Microsoft.AnalysisServices.OperationException: Either the user, DEVE\yiwushi, does not have access to the DBAdmin database, or the database does not exist.

   at Securitisation.Utilities.SSAS.UtilSSAS.SSASServerConnect(String sConnectionString)
   at Securitisation.Utilities.SSAS.UtilSSAS.ProcessSSASDBByName(String sDBName, String sSSASConnection)
   at BL.TaskProcessService.OperationProvider.ProcessCube(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.TaskProcessService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\TaskProcessService.svc.cs:line 63
